packages/core/src/login/oidc/IOidcOptions.ts

```diff
@@ -33,28 +33,31 @@ import { IClient } from "./IClient";
 /**
  * @hidden
  */
-export interface ICoreOidcOptions {
+export interface IOidcOptions {
+  /**
+   * The URL of the Solid Identity Provider.
+   */
   issuer: string;
+  /**
+   * The openid-configuration of the issuer.
+   */
   issuerConfiguration: IIssuerConfig;
   client: IClient;
   sessionId: string;
   refreshToken?: string;
-}
-
-/**
- * @hidden
- */
-export interface IAccessTokenOidcOptions extends ICoreOidcOptions {
+  /**
+   * Specify wether the Solid Identity Provider may or may interact with the user.
+   */
+  prompt?: string;
+  /**
+   * True if a dpop compatible auth_token should be requested.
+   */
   dpop: boolean;
+  /**
+   * The URL to which the user should be redirected after authorizing.
```

Comment:
```suggestion
   * The URL to which the user should be redirected after logging in and 
   * authorizing our app to access their Pod.
```

Reply:
I rephrased this slightly in 2f7f4b921cbb20661cc1adccba736312f7e587da, just to be extra clear that logging in refers in this context to the user logging in the IdP